ui/src/app/components/copy-modal.tsx

```diff
@@ -1,18 +1,20 @@
+import * as fp from 'lodash/fp';
 import * as React from 'react';
 
 import { Button } from 'app/components/buttons';
 import { styles as headerStyles } from 'app/components/headers';
 import { Select, TextInput, ValidationError } from 'app/components/inputs';
 import { Modal, ModalBody, ModalFooter, ModalTitle } from 'app/components/modals';
-import {ConceptSet, FileDetail, ResourceType, Workspace} from 'generated/fetch';
+import {CdrVersionListResponse, ConceptSet, FileDetail, ResourceType, Workspace} from 'generated/fetch';
 
 import { Spinner } from 'app/components/spinners';
 import { workspacesApi } from 'app/services/swagger-fetch-clients';
+import {reactStyles, withCdrVersions} from 'app/utils';
 import { navigate } from 'app/utils/navigation';
 import {toDisplay} from 'app/utils/resourceActions';
 import { WorkspacePermissions } from 'app/utils/workspace-permissions';
 
-enum RequestState { UNSENT, ERROR, SUCCESS }
+enum RequestState { UNSENT, COPY_ERROR, SUCCESS }
```

Comment:
renamed for the benefit of the follow-on PR which will have different kinds of errors.  The original is only set for errors after attempting to copy.